src/app/_preview.scss

```diff
@@ -27,6 +27,7 @@ $sg-min-height: 50px;
 $sg-gutter-size: 15px;
 
 body {
+  height: 100vh;
```

Comment:
@LucaMele 
@jackblackCH 
Do we really want to use viewport units, if there is nor real benefit in comparision to `100%` in this case?
https://caniuse.com/#feat=viewport-units

Reply:
@AndyOGo im generally very worried about `vh vw` but as its supported from all our browsers (as u can see in the provided can i use link) AND is "only" the preview, i say its ok to use

Reply:
Well, if you are worried.
And we obviously have no benefit here compared to `100%`.
Let's just use safe code.

Reply:
@AndyOGo and @jackblackCH , safe code is anyway. As is not an issue i let @jackblackCH choose what is best.